src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/utils/CaseDataBuilder.java

```diff
@@ -348,7 +348,7 @@ public CaseDataBuilder withRespondentWithAddress(String respondentName, String a
         return this;
     }
 
-    private Address createAddress(String addressLine1, String addressLine2, String addressLine3,
+    public Address createAddress(String addressLine1, String addressLine2, String addressLine3,
```

Comment:
JavaDoc?

Reply:
oops my mistake this should be private still